imports/api/posts/server/publications.js

```diff
@@ -7,16 +7,17 @@ Meteor.publish('postsLimited', function (options) {
     sort: Object,
     limit: Number
   });
-  return Posts.find({}, options);
+  return Posts.find({approved: true}, options);
 });
 
 Meteor.publish('singlePost', function (slug) {
   check(slug, String);
   return Posts.find({ slug: slug });
 });
 
-Meteor.publish('allPosts', function() {
-  return Posts.find();
```

Comment:
For the publication above the one I deleted, I think we should make it resemble the other `single*` publications.